app/src/main/java/com/example/androidproject/network/endpoints/HomeProviderEndpoint.kt

```diff
@@ -6,5 +6,8 @@ import retrofit2.http.*
 interface HomeProviderEndpoint {
 
     @GET("home-provider/v1/address")
-    suspend fun suggestAddress(@Query("q") query: String, @Query("size") size: Int = 20): List<SuggestedAddressV1>
+    suspend fun suggestAddress
+            (@Query("q") query: String,
```

Comment:
line indentation in this situation should be:
```suggestion
           suspend fun suggestAddress(
               attribute1: String,
               attribute2: String
               ): List<Something>
```
Did you setup your android studio indentation as it was described in project readme in GitHub? If yes, always use  `cmd + option + L` to auto reformat code